webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -166,12 +161,7 @@ const HostDetails = ({
             tabs={tabs}
             router={history}
           >
-            <Tabs
-              style={{
-                width: window.innerWidth - (isNavCollapsed ? 95 : 220),
-              }}
-              activeKey={activeTab}
-            >
+            <Tabs activeKey={activeTab} className="host-details-tabs">
```

Comment:
The dynamic width is calculated for having the tabs scrolling mechanism to work as expected when the menu is collapsed

before:
https://user-images.githubusercontent.com/11807069/146045867-da063265-88c4-46aa-9266-87dd68be6552.mov

with this PR:
https://user-images.githubusercontent.com/11807069/146045918-2836af78-0cb5-4251-8fe3-ab110b779de3.mov

Reply:
Fixed